src/context/AgencyContext.js

```diff
@@ -1,16 +1,67 @@
 import { createContext, useState } from "react";
+import axios from "axios";
+import { APIUrl } from "../config/config";
 
 export const AgencyContext = createContext({});
 
 export default function AgencyContextProvider({ children }) {
-  const [selectedAgency, setSelectedAgency] = useState();
+  const [selectedAgency, setSelectedAgency] = useState({ company_id: 1 });
```

Comment:
Consider getting the selected Company from the url (as a param)
This will work much better when someone accidentally reloads 

Not needed for now - but you can consider this as an improvement later

Also, this is hardcoded to always be `1`? 

Even you do this for testing purposes we will need to remove this before we can ship this.

Reply:
initial state is deleted.
not an agencydetail-page right now...